build/conf.js

```diff
@@ -183,6 +183,11 @@ export default {
     systemBannerSeverity: gulpUtil.env.systemBannerSeverity !== undefined ?
         gulpUtil.env.systemBannerSeverity :
         '',
+    /**
+     * Allows to override disable skip button option on the backend.
+     */
+    enableSkipButton: gulpUtil.env.enableSkipButton !== undefined ? gulpUtil.env.enableSkipButton :
```

Comment:
simplify to `gulpUtil.env.enableSkipButton === true`?

Reply:
The current one is actually correct as we need skip button to be enabled in our dev environments and tests by default. Most of our UI functionality tests rely on dashboard SA privileges.

It only affects our `gulp serve`, `gulp serve:prod` and tasks that rely directly on them, so tests mostly.

So in case someone simply uses `gulp serve` skip option will be visible. To disable it user will have to provide a `enableSkipButton` parameter. `gulp serve --enableSkipButton=false`.

Reply:
would `gulp serve` or `gulp serve:prod` ever be used to actually run the dashboard?

Reply:
@liggitt I don't think so. They are used only for manual tests before creating a PR or to check the changes. Also, they do not deploy container in the cluster but rather start one outside of the cluster.